packages/admin/cms-admin/src/dam/DataGrid/fileUpload/useDamFileUpload.tsx

```diff
@@ -39,16 +40,22 @@ interface FileWithFolderPath extends FileWithPath {
 
 interface UploadFileOptions {
     acceptedMimetypes?: string[];
-    onAfterUpload?: (errorOccurred: boolean) => void;
 }
 
 interface Files {
     acceptedFiles: FileWithPath[];
     fileRejections: FileRejection[];
 }
 
+type ImportSource = { importSourceType: never; importSourceId: never } | { importSourceType: string; importSourceId: string };
+
+interface UploadFileOptions {
```

Comment:
This interface has the same name as `UploadFileOptions` in L41, so it will be merged. But they're two different things. Please rename one accordingly.